Fix all tests on Windows and add run-script for Walkontable watch (#5878)

warpech opened the pull request:
### Context
Continuation of https://github.com/handsontable/handsontable/pull/6128. This fixes https://github.com/handsontable/handsontable/issues/5878

I need this change, because I am using a Windows desktop PC for development. That's because `time npm run build` takes 53s on Windows and 1m21s on macOS.

### How has this been tested?
Run the following scripts on Windows with success:

```
npm run test
npm run test:walkontable
npm run test:walkontable.watch
npm run test:e2e
npm run test:e2e.watch
npm run test:unit
npm run test:unit.watch
```

As well as manually running spec runners in Chrome:

- http://localhost:5000/test/E2ERunner.html
- http://localhost:5000/src/3rdparty/walkontable/test/SpecRunner.html

### Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. https://github.com/handsontable/handsontable/issues/5878
2.
3.

### Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project,
- [ ] My change requires a change to the documentation.

swistach commented on src/3rdparty/walkontable/test/spec/selection.spec.js:
```diff
@@ -155,11 +155,19 @@ describe('Walkontable.Selection', () => {
-    expect(wt.wtTable.getFirstVisibleRow()).toEqual(10);
+
+    let expectedFirstVisibleRow = 10;
+    if (wt.wtSettings.getSetting('scrollbarHeight') >= 17) {
```
And on Windows in MSEdge is `16px`. Maybe it'll be better to calculate `expectedFirstVisibleRow` basing on container height and rows heights?

warpech replied:
Thanks for the suggestion. I have pushed a soution with even cleaner solution in the commit https://github.com/handsontable/handsontable/pull/6187/commits/d32db4736a774afb86eeb081db38eefc716cb512. 

The container size is normalized to have the same inner size disregarding of the scrollbar size.

swistach commented on src/3rdparty/walkontable/test/spec/table.spec.js:
```diff
@@ -116,8 +116,18 @@ describe('WalkontableTable', () => {
+
+    let expectedGetCellOutputForRowsInFirstColumn = [-1, -1, HTMLElement, HTMLElement, HTMLElement, HTMLElement, HTMLElement, HTMLElement, HTMLElement, HTMLElement,
+      HTMLElement, HTMLElement, -2, -2, -2, -2, -2, -2, -2, -2];
+    if (wt.wtSettings.getSetting('scrollbarHeight') >= 17) {
```
It's the same thing as in my comment to `selection.spec.js`.

swistach commented on test/e2e/core/selectAll.spec.js:
```diff
@@ -22,8 +22,8 @@ describe('Core.selectAll', () => {
 
-    hot.view.wt.wtTable.holder.scrollTop = 150;
-    hot.view.wt.wtTable.holder.scrollLeft = 150;
+    hot.view.wt.wtTable.holder.scrollTop = 155;
```
This change is motivated by the different scrollbar width between windows and macOS, isn't it?. I wonder if we can calculate this value based on available space, rows heights etc. as in test specs for selection and table.

warpech replied:
I have pushed a soution with even cleaner solution in the commit a5850d17ab1120d4614c2636ab82ac759063191e.

warpech commented:
I have addressed all comments, except of this one:

> Although, sometimes npm run test:walkontable.watch doesn't run tests until you make change in the code (on macOS).

I have noticed it too, but I don't know what to do about it :(

swistach commented on .config/bin/trigger-on-stdout-change.js:
```diff
@@ -54,7 +54,8 @@ function run(commands, commandToTrigger, spawnOpts) {
   if (commandToTrigger) {
-    ['SIGINT', 'SIGTERM'].forEach(function(signal) {
+    ['SIGINT', 'SIGTERM', 'SIGQUIT'].forEach(function(signal) {
+   //   ['SIGKILL', 'SIGTERM', 'SIGINT', 'SIGQUIT', 'SIGSTP', 'SIGHUP'].forEach(function(signal) {
```
Do we need this line?

warpech replied:
Of course not, thanks for noticing!
